apps/docs-snippets2/src/contracts/deploying-contracts/storage-slots.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable import/first */
+/* eslint-disable import/order */
+import {
+  StorageTestContractFactory,
+  StorageTestContract,
+} from '../../typegend';
+
+const { storageSlots, abi } = StorageTestContract;
+const { bytecode } = StorageTestContractFactory;
+
+// #region full
+// #context import storageSlots from '../your-sway-project/out/debug/your-sway-project-storage_slots.json';
+import { LOCAL_NETWORK_URL, WALLET_PVT_KEY } from '../../env';
```

Comment:
For some reason, I had to leave this line inside of the `#region`, otherwise the generated test file would break. 👀